app/Http/Controllers/GroupController.php

```diff
@@ -766,22 +790,55 @@ public function edit(Request $request, $id)
         ]);
     }
 
-    // TODO: is this alive?
     public function delete($id)
     {
-        if (FixometerHelper::hasRole($this->user, 'Administrator')) {
-            $r = $this->Group->delete($id);
+        $group = Group::where('idgroups', $id)->first();
+
+        $name = $group->name;
+
+        if (FixometerHelper::hasRole(Auth::user(), 'Administrator') && $group->canDelete()) {
+            $r = $group->delete($id);
+            $r = true;
             if ( ! $r) {
-                $response = 'd:err';
+                return redirect('/user/forbidden');
```

Comment:
Does this route still exist?

Reply:
Yes - see https://restarters.net/user/forbidden